Update surveys BE (24/1)

LisaHermans opened the pull request.

LisaHermans committed:
Update surveys BE (24/1)

phev8 commented:
Hi,
this tells me there are merge conflicts in both the intake and weekly files, can you have a look please how to resolve them? I don't know which version is the correct one.

phev8 commented:
@danielapaolotti @m-delfino do you know maybe if the repos between InfectieradarBE and this were in sync?

m-delfino commented:
@phev8 I think they were not, I'll take a look at this as soon as possible, I also made some changes here:

https://github.com/influenzanet/study-manager-app/pull/141

and I have to be sure they remain in place

LisaHermans committed:
Update inf-weekly.ts

m-delfino commented:
@LisaHermans @phev8 I'm sorry but this pull request originates from a commit which is too old and I do not feel comfortable resolving the merge conflict risking to break BE surveys in the `dev` branch on which, meanwhile, landed:

- fixes to the common pool
- IT surveys

together with @cturbelin we are trying to rework the common surveys to make them more easily shared among participating countries so it's best if we freeze contributions to the `dev` branch until that work is finished.

md-isi committed:
update sass

md-isi committed:
add case-editor-tools

needed for easier rules creation

md-isi committed:
surveys cleanup

md-isi committed:
add belgium rules

md-isi committed:
easier rules export

LisaHermans committed:
update surveys 12/09

LisaHermans committed:
update 13/09

md-isi committed:
re-assign priority intake after 1y

sebastiaanweytjens committed:
Survey follow up fix

sebastiaanweytjens committed:
Merge branch 'dev' of https://github.com/InfectieradarBE/study-manager-app into dev

roelsprimont committed:
Vaccine survey - november 2023

roelsprimont committed:
Changed minimum age to 18

md-isi committed:
update study rules for vaccination

- handle child flag
- handle vaccination completed flag

md-isi committed:
add vaccination custom rules

- cleanup
- reset

md-isi committed:
fix rules buttons in ui

md-isi committed:
update case dependencies

this will actually switch to the new survey model (v1.2)

md-isi committed:
first contacts survey stub

md-isi committed:
add rule for updating the isChild flag

roelsprimont committed:
add vaccination flag & update rules accordingly

roelsprimont committed:
Fix oversight, if participant already has vaccine survey assigned

roelsprimont committed:
Add custom rule and move assignment to custom rule instead of timer

roelsprimont committed:
Separate disabling and enabling of vaccination survey custom rules

roelsprimont committed:
add vaccination eligibility check

md-isi committed:
update rules to the latest survey-engine

md-isi committed:
tsc: exclude external path

md-isi committed:
add prettier dependency

md-isi committed:
update contacts survey

md-isi committed:
add contacts survey rules

md-isi committed:
fix survey export format

roelsprimont committed:
Basic symptomsFeedback implementation

roelsprimont committed:
Delete custom gender rule and fix update rule

md-isi committed:
add be languages

md-isi committed:
use an empty versionId instead of 'what?'

md-isi committed:
Merge remote-tracking branch 'dev' into contacts-survey

md-isi committed:
update contact-survey subrepo

this will break the checkout of commit 96ce057 since the contacts survey
repo was force pushed

roelsprimont committed:
Update vaccination survey 2024-2025

roelsprimont committed:
Removal of vaccinationSurveyActive flags

roelsprimont committed:
Add new timer rule for vaccination survey assignment

roelsprimont committed:
Check survey active first and make resetVaccination more strict

roelsprimont committed:
Remove vaccinationCompleted flag when running reset rule

roelsprimont committed:
Disable vaccination survey

roelsprimont committed:
Update vaccination survey 2025-2026

roelsprimont committed:
Set vaccination survey to active and update vaccinationCompleted flag to no on re-activation

roelsprimont committed:
Fix up package.json

gf-isi committed:
added heatwave study

gf-isi committed:
Merge branch 'newer-node-compatibility' into dev

gf-isi committed:
updated submodule

gf-isi committed:
fix problem with intake submission rules

roelsprimont committed:
custom rule for reassinging participants to contact survey
